### PR TITLE
prepare for 2.16.0 release

### PR DESCRIPTION
**Pull request by mikehardy**
- using May 1 as placeholder for whatever the date is it actually will launch
- looking forward to smaller releases that don't require so much changelog work

### CODE REVIEW

File: changelog.asc [hunk 1]
```diff
@@ -2,6 +2,76 @@
...
+** Faster syncing and importing
+** AnkiDroid no longer needs permission to access “All Files”!
+** ...But you lose access to data on uninstall
+** Sync, export/import, check media/database disabled during migration
+** Manage Space Activity in System Settings
+** Added “Backup Reminder” prompt
+** https://github.com/ankidroid/Anki-Android/wiki/Storage-Migration-FAQ[More info in our FAQ]
+* A huge number of crash fixes, too many to mention
+* “Auto” Day/Night theming to follow system, Android 13+ themed icon
+* Added Anki “New Anki Backend option in advanced settings - will be default in 2.17
```
Comment (dae) [inline changelog.asc:20]: Unmatched "

File: changelog.asc [hunk 2]
```diff
@@ -2,6 +2,76 @@
...
+** Add Preference for double-tap time interval
+** “Insert Field” in Card Template Editor option
+** Add "Allow sync on metered connections" option
+** New “About” section
+** “Developer Options” is now available
+* Flags added: Pink, turquoise & purple flags as in Desktop
+* Drawing Activity
+* Custom Sync Server improvements
+** Updated to match new Anki Desktop built-in sync server abilities
+** Use different server for collection versus media
```
Comment (dae) [inline changelog.asc:44]: I believe that got reverted

Comment (mikehardy) [reply changelog.asc:44]: oh yeah - I think you're right. It was a big part of the original commit series from oakkitten but when you came and reorganized things you re-combined

File: changelog.asc [hunk 3]
```diff
@@ -2,6 +2,76 @@
 :sectanchors:
 = AnkiDroid Changelog
 
+== Version 2.16.0 (20230501)
```
Comment (mikehardy) [inline changelog.asc:5]:
```suggestion
== Version 2.16.0 (20230726)
```

File: changelog.asc (hunk 2 above)
Comment (mikehardy) [inline changelog.asc:44]:
```suggestion
```

**Comment by dae** [comment]
Thanks for taking the time out of your travel for this Mike. I can see the Play store listing available versions as "varies with device", so the app suddenly becoming available for anyone who wants it was probably too much to hope for. I've posted a question in the forums anyway, to see what other users see: https://forums.ankiweb.net/t/ankidroid-2-16-release-rollout/32495

**Pull request merged**